src/Shared/testassets/CreateHostBuilderPatternTestSite/Program.cs

```diff
@@ -9,7 +9,7 @@ public class Program
     {
         public static void Main(string[] args)
         {
-            var webHost = CreateHostBuilder(args).Build();
+            using (var webHost = CreateHostBuilder(args).Build()) { }
```

Comment:
This is causing a strange compiler error on the CI. Run build.cmd locally and see if you can track it down.
```
Program.cs(12,20): error CS1674: 'IHost': type used in a using statement must be implicitly convertible to 'System.IDisposable' or implement a suitable 'Dispose' method. [F:\workspace\_work\1\s\src\Shared\testassets\CreateHostBuilderPatternTestSite\CreateHostBuilderPatternTestSite.csproj]
[error]Program.cs(12,20): error CS1674: 'IHost': type used in a using statement must be implicitly convertible to 'System.IDisposable' or implement a suitable 'Dispose' method.
Program.cs(12,20): error CS1674: 'IHost': type used in a using statement must be implicitly convertible to 'System.IDisposable' or implement a suitable 'Dispose' method. [F:\workspace\_work\1\s\src\Shared\testassets\CreateHostBuilderPatternTestSite\CreateHostBuilderPatternTestSite.csproj]
[error]Program.cs(12,20): error CS1674: 'IHost': type used in a using statement must be implicitly convertible to 'System.IDisposable' or implement a suitable 'Dispose' method.
```

Edit: I see the issue. This test is testing the class structure of Program.CreateHostBuilder, but it uses mocks for the types.
https://github.com/aspnet/Extensions/blob/ae950eba800c2c70da9a57bf183bea75f0849d31/src/Shared/testassets/MockHostTypes/IHost.cs
You can just revert this file.